shared/reducers/tracker2.tsx

```diff
@@ -49,8 +49,6 @@ export default Container.makeReducer<Actions, Types.State>(initialState, {
     const d = getDetails(draftState, username)
     d.bio = action.payload.bio
     d.blocked = action.payload.blocked
-    d.followersCount = action.payload.followersCount
```

Comment:
Will be set later when follows are loaded

Reply:
(meanwhile there's a spinner)

Reply:
but we don't ever load these in the tracker right?

Reply:
Good point. Can we hide this in the tracker then?

Reply:
this was the motivation for having the count and the list being 2 things. i'd ask cecile